File: slingcessories_mobile_flutter/lib/models/create_slingshot_dto.dart
class CreateSlingshotDto {
  final int year;
  final String model;
  final String color;
  final String userId;

  CreateSlingshotDto({
    required this.year,
    required this.model,
    required this.color,
    required this.userId,
  });

  Map<String, dynamic> toJson() {
    return {
      'year': year,
      'model': model,
      'color': color,
      'userId': userId,
    };
  }
}

File: Slingcessories.Mobile.Maui/Models/CreateSlingshotDto.cs
namespace Slingcessories.Mobile.Maui.Models;

public class CreateSlingshotDto
{
    public int Year { get; set; }
    public string Model { get; set; } = string.Empty;
    public string Color { get; set; } = string.Empty;
    public string UserId { get; set; } = string.Empty;
}
